integration-cli/docker_api_containers_test.go

```diff
@@ -138,11 +138,7 @@ func TestContainerApiStartVolumeBinds(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	bindPath, err := ioutil.TempDir(os.TempDir(), "test")
-	if err != nil {
-		t.Fatal(err)
-	}
-
+	bindPath := path.Join("/tmp", fmt.Sprintf("test.%s", makeRandomString(10)))
```

Comment:
I like the idea for some kind of utility to prevent people tripping over this (something like `daemon.filepath()`?) then no comments would be required, and it would keep working if eventually a Windows Daemon arrives (just thinking out loud here :))